scripts/fires/wfigs_data.py

```diff
@@ -195,13 +237,13 @@ def get_parent_fire(x):
     df["Location"] = df.apply(get_place, axis=1)
     df["FIPS"] = df.apply(get_fips, axis=1)
     df["Location"] = df["Location"].fillna(df["FIPS"])
-
+    df.loc[df["Location"] == '', 'Location'] = df["FIPS"]
     df["IrwinID"] = df["IrwinID"].apply(lambda x: x.lower())
-    df["dcid"] = df["IrwinID"].apply(lambda x: "fire/IRWINID-%s" % x)
+    df["dcid"] = df["IrwinID"].apply(lambda x: "fire/irwinId/%s" % x)
```

Comment:
line 208 has `fire/IrwinId` vs `fire/irwinId` here. which should it be? consider using a constant to avoid this error.

Reply:
done